tests/bluetooth/host/buf/bt_buf_get_evt/src/main.c

```diff
@@ -92,14 +92,18 @@ static struct net_buf_pool *get_memory_pool(bool discardable)
  */
 ZTEST(bt_buf_get_evt_default_events_returns_not_null, test_returns_not_null)
 {
-	static struct net_buf expected_buf;
+	const size_t user_data_size = sizeof(struct bt_buf_data);
+	uint8_t expected_buf_data[user_data_size];
+	struct net_buf *expected_buf = (struct net_buf *)expected_buf_data;
```

Comment:
how does this work? you're casting a size 1 array to a struct net_buf (with much more than 1 byte).

Reply:
I think I made a mistake. `expected_buf_data` should be `sizeof(struct net_buf) + user_data_size`.